spec/src/main/asciidoc/bulkhead.asciidoc

```diff
@@ -71,8 +71,19 @@ public Future<Connection> serviceA() {
 
 When using the thread pool approach, when a request cannot be added to the waiting queue, `BulkheadException` will be thrown.
 
+==== Lifecycle
+
+Unlike some other fault tolerance strategies, bulkhead needs to store some data between invocations: the number of currently running executions, or the queue of waiting executions.
```

Comment:
Delete "Unlike some other fault tolerance strategies," as Circuit Breaker has the same stategy.